app/views/admin/events/show.html.erb

```diff
@@ -17,16 +46,38 @@
     </h2>
     <div id="eventDetailsCollapse" class="accordion-collapse collapse show" aria-labelledby="eventHeading" data-bs-parent="#accordionExample">
       <div class="accordion-body">
-        <p>Ticket Sale Start Date: <%= @event.ticket_sale_start_date %></p>
-        <p>Theme: <%= @event.theme %></p>
-        <p>Theme Details: <%= simple_format(@event.theme_details) %></p>
-        <p>Theme Image Url: <%= @event.theme_image_url %></p>
-        <p>Location: <%= @event.location %></p>
-        <p>Maps Location Url: <%= @event.maps_location_url %></p>
-        <p>Event Timings: <%= simple_format(@event.event_timings) %></p>
-        <p>Further Information: <%= simple_format(@event.further_information) %></p>
-        <p>Ticket Price Information: <%= simple_format(@event.ticket_price_info) %></p>
-        <p>Ticket Information: <%= simple_format(@event.ticket_information) %></p>
+        <p>Ticket Sale Start Date: <%= @event.ticket_sale_start_date.present? ? @event.ticket_sale_start_date.strftime("%d/%m/%Y at %I:%M:%S %p") : 'No set sale date' %></p>
```

Comment:
This could be cleaned up considerably if we made our own date format? https://api.rubyonrails.org/classes/DateTime.html#method-i-to_formatted_s

Reply:
Not the best name but we now have `:decom_standard` for a time format

Reply:
V nice!